app/keepers.go

```diff
@@ -163,6 +165,46 @@ func InitStakingKeeper(appCodec codec.Codec, keys map[string]*sdk.KVStoreKey, ke
 	return &stakingK
 }
 
+// expandPath resolves paths containing the "~" character to the user's home directory.
+func expandPath(path string) string {
+	if !strings.HasPrefix(path, "~/") {
+		return path
+	}
+
+	homeDir := funcs.Must(os.UserHomeDir())
+
+	return filepath.Join(homeDir, path[2:])
+}
```

Comment:
why do we need this? Generally the shell should already resolve that

Reply:
there's a chance that this is passed through someone's systemd setup or as a quoted string so it doesn't get expanded beforehand. I'm fine with removing it if we're ok with a few nodes potentially running into this